src/translations/en.json

```diff
@@ -209,6 +209,50 @@
     },
     "history": {
       "title": "History"
+    },
+    "snackbar": {
+      "global": {
+        "notYourTurn": "It's not your turn!",
+        "cardFrozen": "That card is frozen! You must wait a turn to play it.",
+        "playFromHand": "You can only play a card that is in your hand",
+        "blockedByQueen": "Your opponent's queen prevents you from targeting their other cards",
+        "blockedByMultipleQueens": "You cannot play a Targeted One-Off (Two, Nine) when your opponent has more than one Queen"
+      },
+      "seven": {
+        "pickAndPlay": "You can only play cards from the top of the deck while resolving a seven."
+      },
+      "draw": {
+        "deckIsEmpty": "The deck is empty; you cannot draw.",
+        "handLimit": "You are at the hand limit; you cannot draw."
+      },
+      "faceCards": {
+        "withoutTarget": "Only Kings, Queens, and Eights may be played as Face Cards without a target"
+      },
+      "points": {
+        "onlyPointCards": "You can only play Ace - Ten cards as points",
+        "numberOnlyForPoints": "You can only play a number card as points."
+      },
+      "jack": {
+        "stealOnlyWithJack": "You can only steal your opponent's points with a jack",
+        "stealOnlyPointCards": "You can only steal your opponent's point cards",
+        "noJackWithQueen": "You cannot use a Jack while your opponent has a Queen."
+      },
+      "scuttle": {
+        "rankTooLow": "You can only scuttle if your card's rank is higher, or the rank is the same, and your suit is higher (Clubs < Diamonds < Hearts < Spades)",
+        "mustUsePointCard": "You can only scuttle with an ace through ten",
+        "mustTargetPointCard": "You can only scuttle a card your opponent has played for points"
+      },
+      "oneOffs": {
+        "scrapIsEmpty": "You can only play a 3 ONE-OFF if there are cards in the scrap pile",
+        "opponentHasNoCards": "You cannot play a 4 as a ONE-OFF while your opponent has no cards in hand",
+        "sevenWithEmptyDeck": "You can only play a 7 as a ONE-OFF if there are cards in the deck",
+        "sevenWithOneCardInDeck": "You cannot play a 7 as a ONE-OFF if it is the last card in the deck",
+        "cantPlayWithoutATarget": "You cannot play that card as a ONE-OFF without a target",
+        "blockedByQueen": "Your opponent's queen prevents you from targeting their other cards",
+        "blockedByMultipleQueens": "You cannot play a Targeted One-Off (Two, Nine) when your opponent has more than one Queen",
```

Comment:
Last nitpick:these are the same as the ones in the game.snackbar.global block, right? Let's remove these and use those instead